src/index.html

```diff
@@ -1,13 +1,54 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html lang="en" class="page">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+
+  <body class="page-content">
+    <div class="card" data-qa="card">
+      <img
+        src="images/imac.jpeg"
+        alt="card"
+        class="card__img"
+      >
+      <p class="card__name" >Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</p>
+      <p class="card__code">Код товара: 195434</p>
+
+      <div class="card__review">
+        <div class="stars stars--4">
+          <div class="stars__star">
+          </div>
+          <div class="stars__star">
+          </div>
+          <div class="stars__star">
+          </div>
+          <div class="stars__star">
+          </div>
+          <div class="stars__star">
+          </div>
+        </div>
+        <p class="reviews">Отзывов: 5</p>
```

Comment:
on your card should be only 2 BEM blocks: card and stars, all the others are just card elements